Front-End Work Update

zbirmingham opened the pull request:
Cleaned up JavaScript and CSS, implemented partials template on all pages and created FAQ Form Validation

zbirmingham committed:
Revert "Revert "Front-End Work""

zbirmingham commented:
@young @obraafo would you mind taking a look at this before I merge this

young commented on css/style_updated.css:
```diff
@@ -0,0 +1,859 @@
+}
+
+.selectable_result:hover {
+	background-color: #F15A24 !important;
```
You shouldn't use '!important' unless you absolutely have to

young commented on index.html:
```diff
@@ -49,17 +50,14 @@
+			text-transform: lowercase;
+			z-index: 2;
 			-webkit-backface-visibility: hidden;
 			-webkit-transform: translateZ(2);
```
Should just use transform: translateZ(2); without the prefix. It's pretty well [supported](http://caniuse.com/#search=translate).

young commented on index.html:
```diff
@@ -81,53 +78,8 @@
 			padding-right: 10px;
 			-webkit-transform: translateZ(2);
```
same

young commented on js/business-faq.js:
```diff
@@ -0,0 +1,14 @@
+
+$(document).ready(function() {
+	loadPartials();
+	console.log(location.pathname);
```
should remove console.log()

young commented on js/faq_form_validation.js:
```diff
@@ -0,0 +1,78 @@
+//Checks whether form is open and/or empty
+function check_form() {
```
Minor note: Javascript convention is to use camelcase for function names such as checkForm(). Python convention is to use underscores.

zbirmingham commented:
Ok, I'll make the changes, my major concern is that I am deleting project dependent files.

zbirmingham committed:
Update .gitignore

Removed composer.lock

Commit:
Made edits requested

!important tags removed from style_updated.css
all JavaScript functions renamed with camel case
mysql.ini tracked as well as composer.lock

zbirmingham commented:
@young and @obraafo, let me know what you guys think. I made some changes, if you guys are good. I'll commit it to master.

young commented:
Ship it

zbirmingham commented:
@young thanks for taking a look
